src/Containers/ContributionForm/ContributionForm.js

```diff
@@ -39,9 +38,28 @@ function ContributionForm(props) {
           zip: zip || null,
           phone: phone || null
         }
-        console.log(data)
         setError(null)
         sendRequest(data)
+      } else if(firstName &&
+          lastName &&
+          insurances &&
+          state &&
+          profession) {
+            const data = {
+              first_name: firstName,
+              last_name: lastName,
+              profession: profession,
+              specialties: specialties || null,
+              insurance: insurances,
+              street: street || null,
+              unit: unit || null,
+              city: city || null,
+              state: state,
+              zip: zip || null,
+              phone: phone || null
+            }
+            setError(null)
+            sendRequest(data)
```

Comment:
We might be able to refactor this a bit, by using the spread operator, so that we don't have to write the data object more than once